swagger-codegen.sh

```diff
@@ -52,3 +52,5 @@ sed -i '/.*if finished_at is None:/,+1 d' "thamos/swagger_client/models/analysis
 sed -i '/.*if reason is None:/,+1 d' "thamos/swagger_client/models/analysis_status_response_status.py"
 sed -i '/.*if started_at is None:/,+1 d' "thamos/swagger_client/models/analysis_status_response_status.py"
 sed -i '/.*if log is None:/,+1 d' "thamos/swagger_client/models/analysis_log_response.py"
+
+black thamos/swagger_client
```

Comment:
developer executing the `swagger-codegen.sh`  and the running the pre-commit run. 
wouldn't that behavior bring out the same outcome?

Reply:
Fair point, forgot about pre-commit. :+1: